spec/helpers/ajax-helper.ts

```diff
@@ -139,7 +139,7 @@ export class MockXMLHttpRequest {
   onerror: (e: ErrorEvent) => any;
   onprogress: (e: ProgressEvent) => any;
   ontimeout: (e: ProgressEvent) => any;
-  upload: XMLHttpRequestUpload;
+  upload: XMLHttpRequestUpload = Object.create(null);
```

Comment:
because https://github.com/Brooooooklyn/rxjs/blob/issue/ajax-upload-progress/src/observable/dom/AjaxObservable.ts#L319 would set `xhr.upload.onprogress` while progressSubscriber not empty.